api/proto/teleport/legacy/types/types.proto

```diff
@@ -7222,6 +7222,15 @@ message AWSICResourceFilter {
     // the supplied regex.
     string name_regex = 2 [(gogoproto.jsontag) = "name_regex,omitempty"];
   }
+
+  // Exclude specifies which AWS resources should be explicitly excluded.
+  oneof exclude {
+    // ExcludeId indicates that a resource should be excluded by exact ID.
```

Comment:
I wonder if we should also rename `include.Id` field to be `include.IncludeID` now.

Reply:
The  `include.Id` for me makes more sense since ` include.IncludeID` have a duplication.